examples/qml_features/CMakeLists.txt

```diff
@@ -77,6 +77,7 @@ if(BUILD_TESTING)
         NAME ${APP_TEST_NAME}
         COMMAND ${TEST_CMD}
     )
+    set_property(TEST ${APP_TEST_NAME} PROPERTY ENVIRONMENT "QT_QPA_PLATFORM=offscreen")
```

Comment:
note you can also set this in the setup of the QUICK_TEST_MAIN_WITH_SETUP with `qputenv("QT_QPA_PLATFORM", "offscreen");` at the start of the constructor, unsure if that is better or worse here?

And unsure why we didn't enable this before?

Reply:
Keeping it in CMake is a bit more flexible, as you can then still choose the platform after building by running the test executable manually.
So let's go with that for now.